fdm-docs/docs/03-Core concepts/032-Schema.md

```diff
@@ -0,0 +1,210 @@
+|--------------------|------------|--------------------------------------------------------------|
+| **p_app_id**       | `text`      | Primary key for the fertilizer application record.          |
+| **b_id**           | `text`      | References `fields.b_id` (the field where it was applied).  |
+| **p_id**           | `text`      | References `fertilizers.p_id`.                              |
+| **p_app_amount**   | `number`    | Quantity of the fertilizer applied (kg / ha).               |
+| **p_app_method**   | `text`      | How the fertilizer was applied (spread, injected, etc.).    |
+| **p_app_date**     | `date`      | Date the fertilizer was applied.                            |
+| **created**        | `timestamp` | Timestamp when this record was created.                                         |
+| **updated**        | `timestamp` | Timestamp when this record was last updated.                                         |
+
+---
+
+## **`soilAnalysis`**  
+**Purpose**: Captures the results of soil analysis for a particular sampling event.
+
+| Column            | Type        | Description                                                         |
+|-------------------|------------|---------------------------------------------------------------------|
+| **a_id**          | `text`      | Primary key for the soil analysis record.                           |
+| **a_date**        | `date`      | The date the soil analysis was performed.                           |
+| **a_source**      | `text`      | Source or entity that provided the soil analysis data.              |
+| **a_p_al**        | `number`    | Total Phosphate content extracted with Ammonium Lactate (mg P2O5/100 g)  |
+| **a_p_cc**        | `number`    | Phosphor plant available content extracted with CaCl2 (mg P / kg)                           |
+| **a_som_loi**     | `number`    | Soil organic matter content (Loss on Ignition) (%)         |
+| **b_gwl_class**   | `gwlClassEnum`      | Groundwater classification.                                         |
+| **b_soiltype_agr**| `soilTypeEnum`      | Agricultural soil type classification                   |
+| **created**       | `timestamp` | Timestamp when this record was created.                                       |
+| **updated**       | `timestamp` | Timestamp when this record was last updated.         
+
+### `soiltypeEnum`  
+Name: `b_soiltype_agr`  
+Possible values: `moerige_klei`, `rivierklei`, `dekzand`, `zeeklei`, `dalgrond`, `veen`, `loess`, `duinzand`, `maasklei`         
+
+### `gwlClassEnum`  
+Name: `b_gwl_class`  
+Possible values: `II`, `IV`, `IIIb`, `V`, `VI`, `VII`, `Vb`, `-`, `Va`, `III`, `VIII`, `sVI`, `I`, `IIb`, `sVII`, `IVu`, `bVII`, `sV`, `sVb`, `bVI`, `IIIa`     
+
+---
+
+## **`soilSampling`**  
+**Purpose**: Records details of where and how the soil sample was taken, linking it to the field and the corresponding analysis.
+
```

Comment:
do you somewhere defined allowed soil parameters to be measured and a predefined terminology for these properties *e.g. "a_som_loi" or "a_c_rt"? Might be added later

Reply:
Yes, the definitions can be found in the description of table `soilAnalysis`, but only a small set of parameters is currently supported: https://github.com/SvenVw/fdm/blob/1ca5e3a3faf68983920922e17ba9fde5dbb92313/fdm-docs/docs/03-Core%20concepts/032-Schema.md?plain=1#L170